src/http_server.h

```diff
@@ -209,9 +209,8 @@ class HTTPAPIServer : public HTTPServer {
 
     uint32_t IncrementResponseCount();
 
-#ifdef TRITON_ENABLE_TRACING
+    // Only used if tracing enabled
```

Comment:
Just checking if this builds ok without enabled tracing. I don't think that tracing libraries are available, when `TRITON_ENABLE_TRACING` is OFF, so that is why we used wrapper `#ifdef TRITON_ENABLE_TRACING` here

Reply:
Built fine for me, but will triple check and should be caught by L0_build_variants as well. Will let you know after pipeline.